CHANGELOG.md

```diff
@@ -4,19 +4,41 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [Unreleased - 0.18.3] - DATE
+## [Unreleased - 0.19.1] - DATE
+### Added
+### Changed
+### Deprecated
+### Removed
+### Fixed
+### Security
+
+## [0.19.0] - 2022-04-19
 ### Added
 - Added support for certificate renewals after expiry using the claim `allowRenewalAfterExpiry`.
 - Added support for `extraNames` in X.509 templates.
+- Added `armv5` builds.
 - Added RA support using a Vault instance as the CA.
+- Added `WithX509SignerFunc` authority option.
+- Added a new `/roots.pem` endpoint to download the CA roots in PEM format.
+- Added support for Azure `Managed Identity` tokens.
 - Added support for automatic configuration of linked RAs.
+- Added support for the `--context` flag. It's now possible to start the 
+  CA with `step --context=abc` to use the configuration from context `abc`.
```

Comment:
`step-ca --context=abc`?

Reply:
Thx! Fixed in 714b5e61e2951de15c77e1b2fc40140bec739182.